### PR TITLE
Fix/toggle map

### PR DESCRIPTION
**Pull request by alvaroha1**
filter on the works

### CODE REVIEW

File: src/components/SwitchViews.js [hunk 1]
```diff
@@ -45,11 +39,11 @@ class SwitchViews extends Component {
 	}
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = () => ({
 });
```
Comment (stevenspyramid) [inline src/components/SwitchViews.js]: this one is redundant, can you delete it?

**Comment by alvaroha1** [comment]
yeah you were right now gone

File: src/components/Filter.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ class Filter extends Component {
...
 			price:'',
 			size:'',
 		}
 	}
 
-	handleSlidersChange = (event) => {
-    const target = event.target;
-    const value = target.value;
+	handleSlidersChange = (e) => {
+		console.log(e);
```
Comment (arol) [inline src/components/Filter.js]: Remove every console log and commented code before ~~pushing~~ creating a pull request.

File: src/components/ItemList.js [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 
-// function ItemList({ItemList}){
-// const listItems = ItemList.map((home)=>
-// <Item url={home.url} thumbnail={home.thumbnail} 
-// 		pricePerSquareMeter={home.pricePerSquareMeter}
-// 		price={home.price} estimatedPrice={home.estimatedPrice}
-// 		 />
+function ItemList({ItemList}){
+const listItems = ItemList.map((home)=>
+<Item url={home.url} thumbnail={home.thumbnail}
```
Comment (arol) [inline src/components/ItemList.js]: Indent correctly.

**Pull request merged**